packages/backend/src/services/UserService.ts

```diff
@@ -44,7 +44,7 @@ import {
 import { randomInt } from 'crypto';
 import { uniq } from 'lodash';
 import { nanoid } from 'nanoid';
-import { requestNewAccessToken } from 'passport-oauth2-refresh';
+import refresh from 'passport-oauth2-refresh';
```

Comment:
this change was for fixing an `any` warning. I'd like to know what's wrong with the change